test/import/ensureConfiguration.test.ts

```diff
@@ -17,20 +17,37 @@ describe('ensureConfiguration', () => {
     loggerSpy.reset();
   });
 
-  it('should return the path to a pre-existing config.yaml', () => {
-    const tank = path.join(__dirname, 'fixtures', 'existing-config-yaml');
+  it('should return the path to a pre-existing sushi-config.yaml', () => {
+    const tank = path.join(__dirname, 'fixtures', 'existing-sushi-config-yaml');
     const configPath = ensureConfiguration(tank);
-    expect(configPath).toBe(path.join(tank, 'config.yaml'));
+    expect(configPath).toBe(path.join(tank, 'sushi-config.yaml'));
+    // Generating a config causes a warning, so there should be NO warnings in this case
+    expect(loggerSpy.getAllLogs('warn')).toHaveLength(0);
+  });
+
+  it('should return the path to a pre-existing sushi-config.yml', () => {
+    const tank = path.join(__dirname, 'fixtures', 'existing-sushi-config-yml');
+    const configPath = ensureConfiguration(tank);
+    expect(configPath).toBe(path.join(tank, 'sushi-config.yml'));
     // Generating a config causes a warning, so there should be NO warnings in this case
     expect(loggerSpy.getAllLogs('warn')).toHaveLength(0);
   });
 
-  it('should return the path to a pre-existing config.yml', () => {
+  it('should return the path to a pre-existing config.yaml and log a warning', () => {
+    const tank = path.join(__dirname, 'fixtures', 'existing-config-yaml');
+    const configPath = ensureConfiguration(tank);
+    expect(configPath).toBe(path.join(tank, 'config.yaml'));
+    expect(loggerSpy.getAllLogs('warn')).toHaveLength(1);
+    expect(loggerSpy.getLastMessage('warn')).toMatch(/Use of config\.yaml is deprecated/);
+  });
+
+  it('should return the path to a pre-existing config.yml and log a warning', () => {
     const tank = path.join(__dirname, 'fixtures', 'existing-config-yml');
     const configPath = ensureConfiguration(tank);
     expect(configPath).toBe(path.join(tank, 'config.yml'));
     // Generating a config causes a warning, so there should be NO warnings in this case
```

Comment:
Very  minor, but I don't think we need this comment any more.

Reply:
We don't need that comment any more, so it is no more in 0deac15